base16-theme.el

```diff
@@ -112,8 +98,11 @@ user's customized variables."
 	(cond
 	 ((string= (symbol-name value) "base16-settings-mode-line-box")
```

Comment:
I *think* this could be done with pcase, similar to below. It would be a bit cleaner.

Reply:
If we go with the alternate method of name translation, this wouldn't be needed